tests/test_enterprise/api/test_views.py

```diff
@@ -7624,6 +7680,7 @@ def test_successful_assign_learners_to_group(self):
                 enterprise_customer_user__isnull=True
             )
         ) == 10
+        assert mock_send_group_membership_invitation_notification.call_count == 20
```

Comment:
nit: it'd be good to make some assertion about what arguments this task was called with.